optimum/exporters/tasks.py

```diff
@@ -80,6 +74,22 @@ def get_diffusers_tasks_to_model_mapping():
 
     tasks_to_model_mapping = {}
 
+    try:
+        from diffusers.pipelines.auto_pipeline import (
+            AUTO_IMAGE2IMAGE_PIPELINES_MAPPING,
+            AUTO_INPAINT_PIPELINES_MAPPING,
+            AUTO_TEXT2IMAGE_PIPELINES_MAPPING,
+        )
+    except Exception as e:
+        # Sometimes a user might have two incompatible versions of transformers and diffusers
+        # resulting in a transformers model export failing because of diffusers import.
+        logger.warning(
+            "`get_diffusers_tasks_to_model_mapping` failed to import diffusers with the error below."
+            "Please make sure you have diffusers installed and compatible with your transformers version.\n"
+            f"{e}"
+        )
+        return tasks_to_model_mapping
```

Comment:
@helena-intel @echarlaix this part should solve the issue you were seeing when incompatible diffusers and transformers are available.